app/assets/stylesheets/vendor/video-js.css

```diff
@@ -798,8 +798,8 @@ body.vjs-full-window {
   cursor: pointer;
   padding: 0;
   margin: 0 0.45em 0 0.45em;
-  background-color: #73859f;
-  background-color: rgba(115, 133, 159, 0.5); }
+  background-color: #004D40;
```

Comment:
I feel you.
I would normally, if it was a **.scss** file.
But this is a normal .css file, so doesn't accept _$variables_.

<img width="413" alt="screen shot 2017-05-08 at 10 21 15 am" src="https://cloud.githubusercontent.com/assets/18417244/25796054/5d06efc6-33d8-11e7-91e0-d2e2c5efa930.png">

**with variables**
<img width="637" alt="screen shot 2017-05-08 at 10 21 23 am" src="https://cloud.githubusercontent.com/assets/18417244/25796065/6650fe8c-33d8-11e7-8c22-3872b27654e0.png">

**without variables**
<img width="685" alt="screen shot 2017-05-08 at 10 22 06 am" src="https://cloud.githubusercontent.com/assets/18417244/25796067/6994de92-33d8-11e7-8419-f174146e30e3.png">